start-linux.sh

```diff
@@ -24,6 +24,7 @@ echo 'DATA_DIR=./data' >> .env
 # run build
 pelias compose pull
 pelias elastic start
+sleep 30
```

Comment:
We should define why the sleep is 30 sek

Reply:
Done